### ISSUE TITLE
size analysis

### ISSUE THREAD

**Issue by Fly-dream12** [gfjiangly/cvtools#2]
Could you give a detailed instruction about how to draw the hist of size distribution?

**Comment by gfjiangly** [comment]
```python
import cvtools

img_prefix = '/your_data/images'  # images folder
ann_file = '/your_data/dataset.json'  # your coco data format json
coco_analysis = cvtools.COCOAnalysis(img_prefix, ann_file)
coco_analysis.stats_size_per_cat('/out/analysis/size_per_cat_data.json')
```
Use this [html page](https://github.com/gfjiangly/cvtools/blob/master/cvtools/label_analysis/g2/size_per_cat.html) to visually display the output json, refer to the output json in the html page.

![image](https://user-images.githubusercontent.com/38396233/71608152-d7577a80-2bb9-11ea-8f04-5a125057dd27.png)

**Comment by Fly-dream12** [comment]
Yeah, i have got the json output result and just don't know how to dispaly.
